### PR TITLE
Search scraped data for search query, depending on filter.

### PR DESCRIPTION
**Pull request by gannerr**
Search scraped data for search query, depending on filter.

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -25,9 +25,27 @@ let _map;
...
 const BLACK_OWNED = 'black-owned';
 
+function fetchBusinessNames() {
+  fetch('/businessNames').then(response => response.json()).then((restaurantNames) => {
+    for(let name of restaurantNames) {
+      scrapedSmallBusinesses.add(name);
+    }
+  });
+
+  fetch('/black-owned-restaurants-data').then(response => response.json()).then((restaurantNames) => {
```
[the remark this answers is not in the document]
Comment (gannerr) [reply capstone/src/main/webapp/script.js:40]: yeah

File: capstone/target/portfolio-1/script.js [hunk 2]
```diff
@@ -97,75 +117,86 @@ function createMap() {
...
-        draggable: true,
-        animation: google.maps.Animation.DROP,
-        position: {lat: restaurant.latitude, lng: restaurant.longitude},
-        zIndex: restaurant.zIndex
-    });
-    marker.addListener('click', toggleBounce);
-    marker.addListener('click', displayPanel(restaurant.name));
-    
-    marker.setMap(map);
+function callback(results, status) {
```
[the remark this answers is not in the document]
Comment (gannerr) [reply capstone/target/portfolio-1/script.js:134]: no difference I think I did something wonky I'll delete this one

[the remark this answers is not in the document]
Comment (gannerr) [reply capstone/target/portfolio-1/script.js:134]: nothing, this seems to be something the target folder generates?

File: capstone/target/maven-archiver/pom.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Tue Jul 28 23:19:17 UTC 2020
```
[the remark this answers is not in the document]
Comment (gannerr) [reply capstone/target/maven-archiver/pom.properties:2]: yep

File: capstone/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 4]
```diff
@@ -1,2 +1,3 @@
 /home/andrewgan/step196-2020/capstone/src/main/java/com/google/sps/servlets/DataServlet.java
+/home/andrewgan/step196-2020/capstone/src/main/java/com/google/sps/servlets/BlackOwnedRestaurantsDataServlet.java
```
[the remark this answers is not in the document]
Comment (gannerr) [reply capstone/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:2]: sure

File: capstone/target/portfolio-1/script.js [hunk 5]
```diff
@@ -12,11 +12,31 @@
...
+  if (document.getElementById('filters-form')) {
+    getInputFilters();
+  }
+});
+
+let _map;
+let _center;
+let _showSmallBusiness = false;
+let _showBlackOwnedBusiness = false;
+let scrapedSmallBusinesses = new Set();
```
[the remark this answers is not in the document]
Comment (gannerr) [reply capstone/target/portfolio-1/script.js:28]: sure

File: capstone/target/portfolio-1/script.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (terencedignon) [reply capstone/target/portfolio-1/script.js]: oh, I was under the impression we were gitignoring this path? LGTMing the PR so I don't block you, but if it is should be gitignored would you mind removing it?

**Pull request merged**